app/Models/Alertgroup.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace App;
+
+use App\Contracts\DBTable;
+use App\Notifications\UserWasRegistered as RegisteredNotification;
+use App\Notifications\PasswordWillReset as PasswordResetNotification;
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Notifications\Notifiable;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+class Alertgroup extends Model
+{
+
+    /**
+     * @var string
+     */
+    protected $table = DBTable::ALERTGROUP;
```

Comment:
@khanhpvrk You do not need to define `$table`. Laravel will automatically map your model to the table name.